### ISSUE TITLE
Learned temperature should clip

### ISSUE THREAD

**Issue by cfoster0** [cfoster0/CLAP#28]
As in the CLIP paper, we should clip the contrastive loss temperature so that the logits are not scaled by more than 100. Should be relatively easy.
